pdc/apps/common/viewsets.py

```diff
@@ -201,11 +201,13 @@ def initial(self, request, *args, **kwargs):
 
     def _check_ordering_keys(self, request):
         ordering_keys = request.query_params.get('ordering')
+        valid_fields = [field.name for field in self.queryset.model._meta.fields]
```

Comment:
I think we discussed, and @simozhan also mentioned. Should it use model's fields? The reasons might be:

1. It could be different name between model and serializer. As defined in `source` of a serializer's field.
2. Serializer's fields could be more than model's. 

Is it still OK to use model's fields?